specs/ngsi-ld_howto.md

```diff
@@ -0,0 +1,189 @@
+# NGSI-LD HowTo
+
+## Introduction
+
+The OMA NGSI information model has been evolved to better support linked data
+(entity relationships), property graphs and semantics (exploiting the
+capabilities offered by JSON-LD). This work has been conducted under the ETSI
+ISG CIM initiative and has been branded as
+[NGSI-LD](https://www.etsi.org/deliver/etsi_gs/CIM/001_099/009/01.01.01_60/gs_CIM009v010101p.pdf).
+The main constructs of NGSI-LD are: Entity, Property and Relationship. NGSI-LD
+Entities (instances) can be the subject of Properties or Relationships. In terms
+of the traditional NGSI data model, Properties can be seen as the combination of
+an attribute and its value. Relationships allow to establish associations
+between instances using linked data. In practice, they are conveyed by means of
+an NGSI attribute, but with a special value (relationship’s object) which
+happens to be a URI which points to another entity. They are similar to the
+“ref” attributes recommended the Data Models guidelines.
+
+Properties and Relationships can be the subject of other Properties or
+Relationships. Thus, in the NGSI-LD information model there are no attribute’s
+metadata, but just “properties of properties” or “properties of relationships”.
+It is not expected to have infinite graphs, and in practice, only one or two
+levels of Property or Relationship “chaining” will happen. Typically, there will
+be one, equivalent to the NGSI metadata abstraction. NGSI-LD Entities are
+represented using JSON-LD, a JSON-based serialization format for Linked Data.
```

Comment:
First instance of JSON-LD should be a link

Reply:
everything fixed at https://github.com/Fiware/dataModels/pull/375/commits/6a4d190ec9a57e47bda230c5fedc2801d63a2a91

Reply:
https://github.com/Fiware/dataModels/pull/375/commits/d94be2f4ffab6861705c2cab1fe2711203a79fcf

Reply:
LGTM